paasta_tools/cli/cmds/logs.py

```diff
@@ -121,18 +152,31 @@ def completer_clusters(prefix, parsed_args, **kwargs):
         return list_clusters()
 
 
+def color_component(component, color_param):
+    # If the color field is iterable try to join
+    # all the colors together, otherwise just apply
+    # the single color
+    try:
+        for color in color_param:
```

Comment:
ah, I see. In that case, I'd prefer a more general 'compose' function:

in ``utils.py``
```
def compose(func_one, func_two):
     def composed(x):
         return func_one(func_two(x))
     return composed
```

then you can do:

```
color_and_bold = compose(bold, yellow)
color_and_bold(line)
```

or even further:
```
italic_bold_color = compose(italic_string, compose(bold, color))
return italic_bold_color("hi!")
```

we can generalise this further to support n functions if you like, but this is a fine start. What do you think?